src/telemetry.ts

```diff
@@ -922,10 +922,10 @@ export class IEventNamePropertyMapping {
         source: 'N/A',
         tags: ['KernelStartup']
     };
-    [Telemetry.DataScienceSettings]: TelemetryEventInfo<{}> = {
-        owner: 'unknown',
-        feature: 'N/A'
-    } as any;
+    // [Telemetry.DataScienceSettings]: TelemetryEventInfo<{}> = {
```

Comment:
@amunger i'm disabling this for now, so that we can have the validation in the PRs
i.e. ensure telemetry is validated in PRs, else we could end up with a siutation where telemetry changes are made and its never validated and we eventually end up with invalid items again